github/repos_rules.go

```diff
@@ -159,16 +159,16 @@ type RequiredWorkflowsRuleParameters struct {
 	RequiredWorkflows    []*RuleRequiredWorkflow `json:"workflows"`
 }
 
-// RuleRequiredCodeScanningTools represents the RequiredCodeScanningTools for the RequiredCodeScanningParameters object.
-type RuleRequiredCodeScanningTools struct {
+// RuleRequiredCodeScanningTool represents the RequiredCodeScanningTools for the RequiredCodeScanningParameters object.
```

Comment:
We might as well go all in and update the comment too, agreed?
Maybe something like this?

```suggestion
// RuleRequiredCodeScanningTool represents a single required code-scanning tool for the RequiredCodeScanningParameters object.
```